mne/viz/_3d.py

```diff
@@ -2365,6 +2365,10 @@ def plot_vector_source_estimates(stc, subject=None, hemi='lh', colormap='hot',
     time_unit : 's' | 'ms'
         Whether time is represented in seconds ("s", default) or
         milliseconds ("ms").
+    glyph : '2darrow' | '3darrow'
+        The type of glyphs to use. If '2darrow', the glyphs belong to a plan
+        and if '3darrow', the glyphs are composite 3d objects (cone +
+        cylinder).
```

Comment:
@GuillaumeFavelier can we just get rid of this option for now? I'm not sure it's going to work given all the scaling issues

Reply:
It should not be difficult to add later. I'll remove it for now because it's not top priority